packages/deployments/contracts/src/cli/init/init.ts

```diff
@@ -454,6 +454,7 @@ export const initProtocol = async (protocol: ProtocolStack, apply: boolean, stag
           // Allowlist watchers in WatcherManager + Connext.
           for (const watcher of protocol.agents.watchers.allowlist) {
             for (const network of protocol.networks) {
+              if (network.chain !== "195") continue;
```

Comment:
shouldn't this be
```suggestion
              if (network.chain === "195") continue;
```

or deleted if we don't want to special case? (we don't really need to leave this in here, as we will be shutting down this testnet-prod env)

Reply:
deleted!